microraiden/microraiden/test/test_eth_ticker.py

```diff
@@ -11,20 +14,30 @@ def test_eth_ticker(
         default_http_client: DefaultHTTPClient,
         sender_privkey: str,
         receiver_privkey: str,
+        monkeypatch: MonkeyPatch
 ):
-    proxy = ETHTickerProxy(receiver_privkey, proxy=empty_proxy) # noqa
-    ticker = ETHTickerClient(sender_privkey, httpclient=default_http_client)
+    def get_patched(*args, **kwargs):
+        body = {
+            'mid': '682.435', 'bid': '682.18', 'ask': '682.69', 'last_price': '683.16',
+            'low': '532.97', 'high': '684.0', 'volume': '724266.25906224',
+            'timestamp': '1513167820.721733'
+        }
+        return jsonify(body)
+
+    monkeypatch.setattr(PaywalledProxyUrl, 'get', get_patched)
```

Comment:
This will work, but please double check if the method is replaced for this test only and we can use the class normally in other tests.

Reply:
The `monkeypatch` fixture takes care of that. That's the beauty of it.

Reply:
cool, let's merge this then